src/invariant/LedgerEntryIsValid.cpp

```diff
@@ -184,10 +184,10 @@ LedgerEntryIsValid::checkIsValid(TrustLineEntry const& tl, uint32 version) const
 std::string
 LedgerEntryIsValid::checkIsValid(OfferEntry const& oe, uint32 version) const
 {
-    if (oe.offerID > INT64_MAX)
+    if (oe.offerID < 0)
```

Comment:
I think this should be `>= 0` as `offerID == 0` is not permitted in the ledger (in operations it indicates create a new offer).